src/app.js

```diff
@@ -2,11 +2,87 @@
  * @param {Object} position
  * @param {Number} position.x
  * @param {Number} position.y
- * 
+ *
  * @returns {Array<position>}
  */
-export function getNearestShops(position) {
-  // code
-  
-  return [];
+import axios from "axios";
+
+const api = "https://blue-bottle-api-test.herokuapp.com/v1"
+const apiHeaders = {'Accept': 'application/json'}
+
+export async function getNearestShops(position) {
+    try {
+        const token = await getTokens()
```

Comment:
I find the name `getTokens` a bit strange. You get one `token`, not multiple `tokens`. On top of that, you're actually `creating` a token, you do a `POST` request here. Could you find a better name for this function?

Reply:
Also, I noticed this is wrapped in a `try...catch` block. I find this [very short video](https://www.youtube.com/shorts/ITogH7lJTyE) helpful. Take a look :+1:.